Changelog.md

```diff
@@ -1,8 +1,52 @@
 # Changelog
 
-## Omigami 0.3.0
-### Summary of changes
-- to be summarized
-  
-### Details
-- to be detailed
\ No newline at end of file
+## Releases
+***
+v0.3.0
+***
+Release Date: 2022-01-10
+
+### New Features
+- **Mirror plot**: Add `SpectraComparisonPlotter().mirror_plot()` to `omigami.plotting` 
+module. It takes two spectra returned from `match_spectra()` and  creates a mirror 
+plot with spectrum 1 on top (blue), and spectrum 2 on the bottom (red).
+- **Caching functionality**: Both `Spec2Vec` and `MS2DeepScore`objects cache 
+successful predictions, also there are 2 new methods:
+  - if predictions for spectra fails, they can be called by `faied_spectra()`.
+  - cache contains successful predictions of spectra, implemented `reset_cache()`
+    to reset it.
+- **Implement SpectrumDataFrameHelper:** Accessible from `omigami.utilities`, 
+this class serves as a helper to create and manipulate Pandas DataFrame versions 
+of Spectrum objects, useful for some plotting functions (such as `mirror_plot()`).
+
+### Bugfixes
+- **More detailed error messages**: Omigami was unable to catch errors from 
+backend during `match_spectra()` call. Omigami properly handles errors and 
+displays them.
```

Comment:
Please add about the bugs regarding precursor_mz/pepmass and ExactMass/parent_mass. They should both be in Jira